src/dndbeyond/base/utils.js

```diff
@@ -485,6 +485,13 @@ async function sendRoll(character, rollType, fallback, args) {
             req.character.settings["custom-roll-dice"] = (req.character.settings["custom-roll-dice"] || "") + " -1d4";
             addEffect(req, "Bane");
         }
+
+        if (req.character.exhaustion && req.character.exhaustion != 0 && req.character.settings["effects-exhaustion"] && 
+            (["attack", "spell-attack"].includes(req.type) && req["to-hit"] || 
+            ["initiative", "ability", "saving-throw", "skill", "death-save"].includes(req.type))) {
+            req.character.settings["custom-roll-dice"] = (req.character.settings["custom-roll-dice"] || "") + ` -${req.character.exhaustion * 2}`;
+            addEffect(req, `Exhaustion (${req.character.exhaustion})`);
```

Comment:
Add a check here for the character sheet version, if 2014, then force disadvantage on ability checks at `exhaustion >= 1` and disadvantage on attacks and saving throws on `exhaustion >= 3`
This way it would support both legacy and 2014 exhaustion rules (if 2014, name the effect as `Legacy Exhaustion (${req.character.exhaustion)`).

Reply:
I will